examples/StateChannelsBar/views/partials/inputSecretKey.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="row form-group">
+    <label for="secretKey" class="col-sm-2">Your secret Key</label>
+    <div class="col col-xs-8">
+        <input type="password" id="secretKey" class="form-control" value="36595b50bf097cd19423c40ee66b117ed15fc5ec03d8676796bdf32bc8fe367d82517293a0f82362eb4f93d0de77af5724fba64cbcf55542328bc173dbe13d33">
```

Comment:
This particular secret key for the `initiator` to open state channel is not working in the demo, causing the browser to throw an error.

```
Error: While calling getAccountByPubkey (pubkey), GET to http://localhost:3001/v2/accounts/ak_zPoY7cSHy2wBKFsdWJGXM7LnSjVt6cn1TWBDdRBUMC7Tur2NQ failed with 404: Account not found
```

Reply:
this is last funded account from the local node... it should be there :) 
`#7 ------------------------------------------------------------
public key: ak_23p6pT7bajYMJRbnJ5BsbFUuYGX2PBoZAiiYcsrRHZ1BUY2zSF
private key: e15908673cda8a171ea31333538437460d9ca1d8ba2e61c31a9a3d01a8158c398a14cd12266e480f85cc1dc3239ed5cfa99f3d6955082446bebfe961449dc48b
Wallet's balance is 10000000000000000
#8 ------------------------------------------------------------
public key: ak_gLYH5tAexTCvvQA6NpXksrkPJKCkLnB9MTDFTVCBuHNDJ3uZv
private key: 6eb127925aa10d6d468630a0ca28ff5e1b8ad00db151fdcc4878362514d6ae865951b78cf5ef047cab42218e0d5a4020ad34821ca043c0f1febd27aaa87d5ed7
Wallet's balance is 10000000000000000
#9 ------------------------------------------------------------
public key: ak_zPoY7cSHy2wBKFsdWJGXM7LnSjVt6cn1TWBDdRBUMC7Tur2NQ
private key: 36595b50bf097cd19423c40ee66b117ed15fc5ec03d8676796bdf32bc8fe367d82517293a0f82362eb4f93d0de77af5724fba64cbcf55542328bc173dbe13d33
Wallet's balance is 10000000000000000

===== Default wallets was successfully funded! =====`

Reply:
So, maybe the instructions for running the example are not quite instructive then. As I ran the docker with `docker-compose up` (by reflex) instead of running `forgae node` for starting up the node instances, resulting to this:

![image](https://user-images.githubusercontent.com/13139371/53480323-a54e2700-3a83-11e9-92e1-567647092cda.png)

When started with `forage node` it this error is gone. Assuming that the accounts are not being prefunded if not started via `forgae`.

Suggestion: More descriptive instructions for running the example :)